cfme-performance/utils/providers.py

```diff
@@ -372,16 +373,58 @@ def add_providers(providers):
 def add_host_credentials(provider):
     """"Adds host credentials to a provider via the REST API"""
     starttime = time.time()
-    ssh_client = SSHClient()
-
-    command = ('p = ExtManagementSystem.find_by_name(\'{}\'); '
-        'for host in p.hosts do; '
+    appliance = cfme_performance['appliance']['ip_address']
+    p_id = get_provider_id(provider['name'])
+    version = get_version()
+    logger.warning('current version is: {}'.format(version))
+    if int(version) < 56:
+        ssh_client = SSHClient()
```

Comment:
Could we pass the ssh_client into the method too since this only occurs serially and we don't need an additional ssh_client object?  (I realize that wasn't the original intent here but lets fix this too)